test/ml/expected/expected-pca-coordinates-2.tsv

```diff
@@ -0,0 +1,4 @@
+algorithm	PC1	PC2
```

Comment:
If these two files are the same except for the PC1 sign, do we want them to be stored separately? Or is it better to track a single file and have the test make a copy and negate PC1?

Reply:
Fair point. I like `negated` better.